document/core/util/macros.def

```diff
@@ -254,6 +254,9 @@
 .. |EOFFSET| mathdef:: \xref{syntax/modules}{syntax-elem}{\K{offset}}
 .. |EINIT| mathdef:: \xref{syntax/modules}{syntax-elem}{\K{init}}
 
+.. |EREFNULL| mathdef:: \xref{syntax/modules}{syntax-element}{\K{ref.null}}
+.. |EREFFUNC| mathdef:: \xref{syntax/modules}{syntax-element}{\K{ref.func}}
```

Comment:
I think it's fine to already call these macros REFNULL and REFFUNC, as in the reftypes proposal.